tests/requirements.txt

```diff
@@ -1,6 +1,7 @@
 pytest
 ragger[speculos]>=1.2.0
 ragger[ledgerwallet]>=1.2.0
+protobuf==3.20.0
```

Comment:
Why do we need to tag protobuf here ?
It's not boilerplate that depends on it but ledgerwallet, and ledgerwallet now specifies the correct version?
Not deployed though, maybe that's why

Reply:
Removed this version